tests/click_tests/test_reset_bip39.py

```diff
@@ -64,7 +64,10 @@ def test_reset_bip39(device_handler: "BackgroundDeviceHandler"):
     reset.confirm_read(debug, "Success")
 
     # Your backup is done
-    debug.press_yes()
+    if debug.model == "T":
```

Comment:
It will (most probably, have not tried) work, but doing `press_right` will also test the button response and its UI change. Being `click` tests, we should use these as much as possible.

We might theoretically use `go_next` here, so it also clicks on `TT` (and also tests the button and UI)

Reply:
Using `go_next` in [031f933](https://github.com/trezor/trezor-firmware/pull/2610/commits/031f933a675f4d22b609d6303d98f4f56fad04a3)